rest/model/task.go

```diff
@@ -330,6 +310,31 @@ func (at *APITask) BuildFromService(t interface{}) error {
 	return nil
 }
 
+func (at *APITask) GetAMI() error {
+	if at.HostId != nil {
```

Comment:
Reminder that it's not if it isn't nil, if it's the string doesn't equal "" (since we always set HostId to a pointer)

Reply:
oops thanks!